source/mobile/mobile-overview.rst

```diff
@@ -1,37 +1,84 @@
----------------------------------------------
+This documentation provides information to help you: 
 
-For quick and easy deployment, you can download the Mattermost mobile apps directly from the Apple App Store and Google Play. Please see our guide on :doc:`installing from the Apple App Store and Google Play Store <mobile-appstore-install>` for more details. 
+- Determine the ideal mobile deployment model for your organization.
+- Understand what’s required to build your own Mattermost Mobile Apps.
+- Deploy Mattermost Mobile Apps.
 
-This option is recommended for:
+.. note::
 
-1. Testing out the mobile applications
-2. Team Edition servers using no push notifications, or push notifications from Mattermost's :ref:`TPNS <tpns>` (Test Push Notification Service)
-3. Enterprise Edition servers using push notifications from Mattermost's :doc:`HPNS <mobile-hpns>` (Hosted Push Notification Service)
+  - Mattermost Cloud customers must use Mattermost pre-built public apps for mobile deployment. 
+  - Some features described in this guide are available only in Mattermost Enterprise Edition.
 
-Build the apps yourself
-------------------------
+Decision Summary
+----------------
 
-To customize the mobile apps, you can fork the open source repository and :doc:`compile your own versions <mobile-compile-yourself>` of the Mattermost mobile applications and Mattermost push proxy server. 
+When planning for a Mobile app deployment, you have two important decisions to make: 
 
-This option is recommended for:
+- What app will you deploy?
+- How will you deploy it?
 
-1. Organizations that want to customize the mobile apps
-2. Team Edition and Enterprise Edition servers that prefer to host their own push proxy server instead of using one of Mattermost's hosted versions
+Mattermost provides official mobile Apps through public app stores including the `Apple App Store <https://www.apple.com/ca/app-store>`__ and the `Google Play Store <https://play.google.com/store>`__. These apps are referred to throughout this documentation as Mattermost's pre-built mobile apps. Using these pre-built apps is the easiest and fastest approach since your users can download the apps from public app stores, or you can use an EMM provider to enforce or restrict specific security policies. See `Using Mattermost’s Pre-Built Apps <https://docs.mattermost.com/mobile/use-prebuilt-mobile-apps.html>`__ to learn more about using Mattermost pre-built apps.
 
-Use an EMM provider with Managed App Configuration
----------------------------------------------------
+If you want to control the look, feel, or capabilities of the Mobile apps, host your own push proxy server, or to maintain full control over app distribution, you can build your own mobile apps and manage your own app distribution. See `Building and Distributing Your Own Custom Mattermost Mobile Apps <https://docs.mattermost.com/mobile/build-custom-mobile-apps.html>`__ to learn more about working with custom apps.
```

Comment:
> If you want to control the look, feel, or capabilities of the Mobile apps

capabilities in what way? are we referring to add custom functionality or remove functionality that already exists?

Reply:
> or to maintain full control over app distribution

☝️ this does not require to have an own build of the apps.

Reply:
Great points! Thanks!